guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Content Strategy K2.0 
+---
+
+## Table of contents
+
+- [Purpose and Audience](#purpose-and-audience)
+- [Information types](#information-types)
+  - [Concept Topics](#concept-topics)
+  - [Task (tutorial) topics](#task-tutorial-topics)
+  - [Reference topics](#reference-topics)
+  - [Troubleshooting topics](#troubleshooting-topics)
+  - [Release Notes](#release-notes)
+- [Graphics](#graphics)
+  - [Architecture and Flow Diagrams](#architecture-and-flow-diagrams)
+  - [Screenshots](#screenshots)
+- [Target Groups](#target-groups)
+  - [Decision maker](#decision-maker)
+  - [Software developer](#software-developer)
+  - [Admin/Operations](#adminoperations)
+- [Documentation Structure](#documentation-structure)
+  - [Overview](#overview)
+  - [Getting Started](#getting-started)
+  - [Deep Dive/Tutorials](#deep-divetutorials)
+  - [Operations](#operations)
+  - [Technical References](#technical-references)
+  - [Glossary](#glossary)
+- [Content Source](#content-source)
+
+## Purpose and Audience
+
+This content strategy focuses on the publicly available documentation under https://kyma-project.io/docs/.
+More documentation may be found in the [Kyma](https://github.com/kyma-project/kyma) GitHub repositories.
+
+The assumed reader of this guide has some basic knowledge of technical writing. To learn more, read the excellent [Istio document about adding documentation](https://istio.io/latest/docs/releases/contribute/add-content/).
```

Comment:
I'm not sure if our devs ever read our guides...? And I'd hope they also have some basic knowledge, otherwise we need to explain all the basic details - write concisely, don't use "please", avoid future tense, recommended sentence structure... :/ 

I agree about the Istio link though, dropping the link.